app/com/linkedin/drelephant/spark/heuristics/ExecutorGcHeuristic.scala

```diff
@@ -102,6 +104,9 @@ object ExecutorGcHeuristic {
       throw new Exception("No executor information available.")
     }
 
+    val isParallelGcEnabled: Boolean = appConfigurationProperties.getOrElse("spark.executor.extraJavaOptions","").contains("XX:+UseParallelGC")
+    val parallelGcRecommendation: String = if (isParallelGcEnabled)  "" else "Enable ParallelGc using spark.executor.extraJavaOptions."
```

Comment:
Change "ParallelGc" to "ParallelGC", in case of copy-paste.

Reply:
Done